hooks/honeybadger/payload.go

```diff
@@ -0,0 +1,48 @@
+package logrus_honeybadger
+
+import "os"
+
+type Payload struct {
+	Notifier *Notifier `json:"notifier"`
+	Error    *Error    `json:"error"`
+	Server   *Server   `json:"server"`
+}
+
+type Notifier struct {
+	Name    string `json:"name"`
+	URL     string `json:"url"`
+	Version string `json:"version"`
+}
+
+type Error struct {
+	Backtrace []*Backtrace `json:"backtrace"`
```

Comment:
this bit isn't used, don't know if its a mistake or on purpose. if on purpose, I'd remove this field and the related `type Backtrace struct`